modules/transcription-service-workflowoperation/src/main/java/org/opencastproject/transcription/workflowoperation/NibityAttachTranscriptionOperationHandler.java

```diff
@@ -142,6 +144,8 @@ public WorkflowOperationResult start(final WorkflowInstance workflowInstance, Jo
           String captionFileType = "vtt";
           mediaPackage = addTranscriptionElementToMediaPackage(zis, captionMimeType, captionIdentifier, captionFileType,
                   mediaPackage, flavor, targetTagOption);
+        } else {
+          workflowInstance.setConfiguration("has-vtt", "false");
```

Comment:
Use the global variable here:
workflowInstance.setConfiguration(HAS_VTT, "false");